cmake/Modules/Findaom.cmake

```diff
@@ -17,19 +17,30 @@
 #=============================================================================
 #
 
-find_package(PkgConfig QUIET)
-if (PKG_CONFIG_FOUND)
-    pkg_check_modules(_AOM aom)
-endif (PKG_CONFIG_FOUND)
+if (AVIF_LOCAL_AOM)
+    find_path(AOM_INCLUDE_DIR
+              NAMES aom/aom.h
+              PATHS ${AVIF_LOCAL_AOM_INCLUDE_DIR}
+              NO_DEFAULT_PATH)
```

Comment:
Is there anyway to pass `NO_DEFAULT_PATH` option to `find_path` only when `AVIF_LOCAL_AOM` is true?